<?xml version="1.0"?>
<runHistory script="_Test5 Create Todo sync">
 <run>
  <RunDate>2016-06-01 12:16:47.047 +0100</RunDate>
  <Status>Success</Status>
  <Duration>4.000000</Duration>
  <Errors>0</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>1</Successes>
  <LogFile>20160601_121647.047/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage></ErrorMessage>
 </run>
 <run>
  <RunDate>2016-06-01 12:17:51.051 +0100</RunDate>
  <Status>Success</Status>
  <Duration>3.000000</Duration>
  <Errors>0</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>1</Successes>
  <LogFile>20160601_121751.051/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage></ErrorMessage>
 </run>
 <run>
  <RunDate>2016-06-01 12:21:44.044 +0100</RunDate>
  <Status>Failure</Status>
  <Duration>3.000000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>0</Successes>
  <LogFile>20160601_122144.044/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage>Connection Failed -- Unable to connect to Server:52.39.154.48 on port:3389
Authentication failed</ErrorMessage>
 </run>
 <run>
  <RunDate>2016-06-01 12:22:18.018 +0100</RunDate>
  <Status>Success</Status>
  <Duration>6.000000</Duration>
  <Errors>0</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>1</Successes>
  <LogFile>20160601_122218.018/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage></ErrorMessage>
 </run>
 <run>
  <RunDate>2016-06-01 12:23:27.027 +0100</RunDate>
  <Status>Success</Status>
  <Duration>7.000000</Duration>
  <Errors>0</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>1</Successes>
  <LogFile>20160601_122327.027/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage></ErrorMessage>
 </run>
 <run>
  <RunDate>2016-06-01 12:35:07.007 +0100</RunDate>
  <Status>Success</Status>
  <Duration>3.000000</Duration>
  <Errors>0</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>1</Successes>
  <LogFile>20160601_123507.007/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage></ErrorMessage>
 </run>
 <run>
  <RunDate>2016-06-01 12:38:07.007 +0100</RunDate>
  <Status>Failure</Status>
  <Duration>7.000000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>0</Successes>
  <LogFile>20160601_123807.007/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage>STUnknownMessage -- ERROR: No Such Command: &#x27;find&#x27;</ErrorMessage>
 </run>
 <run>
  <RunDate>2016-06-01 12:38:59.059 +0100</RunDate>
  <Status>Failure</Status>
  <Duration>3.000000</Duration>
  <Errors>1</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>0</Successes>
  <LogFile>20160601_123859.059/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage>STUnknownMessage -- ERROR: No Such Command: &#x27;find&#x27;</ErrorMessage>
 </run>
 <run>
  <RunDate>2016-06-01 12:42:26.026 +0100</RunDate>
  <Status>Running</Status>
  <Duration>0.000000</Duration>
  <Errors>0</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>0</Successes>
  <LogFile>20160601_124226.026/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage></ErrorMessage>
 </run>
 <run>
  <RunDate>2016-06-01 12:45:21.021 +0100</RunDate>
  <Status>Success</Status>
  <Duration>4.000000</Duration>
  <Errors>0</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>1</Successes>
  <LogFile>20160601_124521.021/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage></ErrorMessage>
 </run>
 <run>
  <RunDate>2016-06-01 13:09:58.058 +0100</RunDate>
  <Status>Success</Status>
  <Duration>8.000000</Duration>
  <Errors>0</Errors>
  <Warnings>0</Warnings>
  <Exceptions>0</Exceptions>
  <Successes>1</Successes>
  <LogFile>20160601_130958.058/LogFile.txt</LogFile>
  <ReturnValue></ReturnValue>
  <ErrorMessage></ErrorMessage>
 </run>
</runHistory>
